### DISCUSSION TITLE
Application.targetFrameRate Forbidden ?

### DISCUSSION THREAD

**Discussion by Deytooh** [naverz/zepeto-studio-kor#1272] [4. General]
안녕하세요.

현제, 월드는 빌드 하고 모바일에 테스트 하면 30fps 까지 올라가는 제한 있네요.
근데 60 fps 까지 올라가고싶어서 Application.targetFrameRate 쓰고 제한 풀려고했는데 할 수 없네요.
다른 방법이 있나요 ? 아니면 60 fps 까지 올라갈 수 있게 만들 예정이 있어요 ?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 모바일 단말에서 FrameRate를 변경하는 것은 위험하여 막아두었습니다.
현재 저희는 30프레임 고정입니다.
참고 부탁 드립니다.

**Reply by Deytooh** [reply]
아 네 ! 
어떻게 위혐해요 ? 🤔

**Reply by Zepeto-TechEvangelist** [reply]
과도하게 프레임을 높일 경우 발열 및 모바일 기기 전력 소모 문제가 발생하기 때문입니다. 
제페토는 글로벌 서비스이기에 최저 사양도 고려해야해서 30프레임으로 제공하고 있습니다. 
감사합니다.

**Reply by Deytooh** [reply]
더 편해서 영어로 할게요.
I dont understand why, because anyway, we can modify the vsync count to override the limitation of 30 fps... and there is already a lot of Worlds in Zepeto that are making the phones so much overheating even with 30 fps limitation ( with not optimization of scripts and / or too many detailed assets / big shaders in their scenes ). So why couldnt we modify this value ?
